src/plugins/kibana_usage_collection/server/collectors/application_usage/saved_objects_types.ts

```diff
@@ -35,13 +35,10 @@ export function registerMappings(registerType: SavedObjectsServiceSetup['registe
     hidden: false,
     namespaceType: 'agnostic',
     mappings: {
+      // Not indexing any of its contents because we use them "as-is" and don't search by these fields
+      // for more info, check on the README.md in this same folder
```

Comment:
```suggestion
      // for more info, see the README.md for application_usage
```
The README might move 😉